apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-events-list/community-events-list.component.ts

```diff
@@ -10,6 +10,8 @@ import { CommunityEventsListDateComponent } from './community-events-list-date/c
 import { CommunityEventsListPublicPageComponent } from './community-events-list-public-page/community-events-list-public-page.component';
 import { debounceTime, filter, map, switchMap, takeUntil } from 'rxjs/operators';
 import { Subject } from 'rxjs';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { SeoService } from 'apps/shared-services/seo.service';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service